.github/workflows/run_update_raster_stats.yml

```diff
@@ -0,0 +1,39 @@
+name: Update exposure raster stats
+
+on:
+  repository_dispatch:
+    types:
+      - trigger_exposure_raster_stats
```

Comment:
This is cool! I haven't used the `repository_dispatch` trigger before. So this is basically listening for the `trigger_exposure_raster_stats` event that is sent from the script above?

Reply:
I actually haven't used it yet either. But yes my understanding is that it's like a `workflow_dispatch`, but more specific. Might be nice to clearly differentiate between triggering the action for testing (via the button on GH) and automated runs triggered by the other action.